<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="hexView">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTreeWidget" name="packetList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>911</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">5</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="HTTPView">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="headerView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>651</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
